test/test_sum_product.py

```diff
@@ -110,58 +110,87 @@ def test_partial_sum_product(impl, sum_op, prod_op, inputs, plates, vars1, vars2
     assert_close(actual, expected)
 
 
-def _expected_hmm_example(sum_op, prod_op, factors, plate_to_step,
-                          global_vars, local_var_dict):
+def _expected_modified_partial_sum_product(
+        sum_op, prod_op, factors, plate_to_step, global_vars,
+        local_var_dict, local_markov_var_dict, markov_to_step):
```

Comment:
Can you explain the function of this new `local_markov_var_dict` argument? It doesn't seem to be used in any of the tests except `test_modified_partial_sum_product_14`.